### ISSUE TITLE
Fonts not changing for Windows 10

### ISSUE THREAD

**Issue by adabo** [neovim/neovim#6278]
- `nvim --version`: v0.1.6-250-gc57a816
- Vim (version: ) behaves differently?
- Operating system/version: Windows 10 10.0.14393
- Terminal name/version:
- `$TERM`:

### Actual behaviour

Changing GuiFont in ginit.vim has no effect

### Expected behaviour

Change font

### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC

```

Download https://ci.appveyor.com/api/projects/neovim/neovim/artifacts/build/Neovim.zip?branch=master&job=Configuration%3A%20MINGW_64

1. Unzip to c:\Prorgram Files\
2. Run nvim-qt.exe
3. edit c:\Users\...\AppData\Local\nvim\ginit.vim
4. Add new line:

`GuiFont Consolas_Regular:h12`

or

`GuiFont Consolas Regular:h12`

**Comment by justinmk** [comment]
try `call GuiFont(...)` instead

**Comment by adabo** [comment]
I'm not sure I'm doing this correctly, because nothing I do is working:

`call GuiFont("Consolas:h12")`

I don't know what the exact name it's looking for. I've tried `call GuiFont("Consolas Regular:h12")` and `call GuiFont("Consolas_Regular:h12")`

And that's not even the font I want to use, but I figured it was a common font that could work.

**Comment by justinmk** [comment]
`:GuiFont Consolas:h12` works for me. 

Consolas is the default already, btw. I don't know what name it needs, it depends on your system.

**Comment by adabo** [comment]
That's odd. It doesn't change anything for me. The default is certainly not Consolas. It looks to be Courier New. Maybe there's a chance it's my init.vim file which uses the code from my _vimrc file I used for GVIM.

[
![image](https://cloud.githubusercontent.com/assets/360653/23922927/7bd41874-08d2-11e7-838b-0c08b00526d2.png)
](url)

**Comment by adabo** [comment]
Tried a clean init.vim and still no change.

**Comment by justinmk** [comment]
If courier isn't the default you're using an old version of neovim-qt.

Did you use the package linked from the wiki? When did you download it?

**Comment by adabo** [comment]
Looks like you figured it out! I don't know where I got my previous build from but the download link from my downloads history looks like this:
`https://appveyorcidata.blob.core.windows.net/neovim-15021/neovim/11618/`

and the new working one:
`https://appveyorcidata.blob.core.windows.net/neovim-15021/neovim/11620/`

Really thanks for your help.

P.S. I do want to point out, though, that I did fill out the prerequisite's in the original post containing the version I was using ;)
